### PR TITLE
feat: 🎸 add KPIProgressItem

### PR DESCRIPTION
**Pull request by justinguo**

### CODE REVIEW

**Review by MarcoEidinger — CHANGES_REQUESTED** [review]
Hi Justin, overall looks good 👍 . From my side, I'd like to ask you to please
- review the code style feedback from Hound/SwiftLint
- add component to README table (https://github.com/SAP/cloud-sdk-ios-fiori#fioriswiftuicore)
- Provide `KPIProgressItemLibraryContent` (conforming to `LibraryContentProvider`) so that app developers can find the the view in the Xcode library

File: Sources/FioriSwiftUICore/DataTypes/KPIItem+DataType.swift [hunk 1]
```diff
@@ -181,3 +202,38 @@ internal struct KPIFormatter {
...
+                    .font(.system(size: 24.0))
+            default:
+                current = current
+                    .font(fractionComponents.isEmpty ? .system(size: 48.0) : .system(size: 36.0))
+            }
+            if index < fullComponents.count - 1 {
+                current = current
+                    .kerning(self.kerning(after: fullComponents[index + 1].type, before: component.type))
+            }
+            text = text + current
```
[the remark this answers is not in the document]
Comment (justinguo) [reply Sources/FioriSwiftUICore/DataTypes/KPIItem+DataType.swift:234]: This one cannot be changed. Shorthand operators are not supported for Text.

[the remark this answers is not in the document]
Comment (justinguo) [reply Sources/FioriSwiftUICore/DataTypes/KPIItem+DataType.swift:234]: Same here.

**Comment by justinguo** [comment]
> Hi Justin, overall looks good 👍 . From my side, I'd like to ask you to please
> 
> * review the code style feedback from Hound/SwiftLint
> * add component to README table (https://github.com/SAP/cloud-sdk-ios-fiori#fioriswiftuicore)
> * Provide `KPIProgressItemLibraryContent` (conforming to `LibraryContentProvider`) so that app developers can find the the view in the Xcode library

Hi Marco, thanks for the feedback. All items are addressed and you can double check.

**Review by MarcoEidinger — APPROVED** [review]
💯 👍

**Pull request merged**